examples/example_pump_data_model/transformations/pump_model-populate-lift_station_pumps_edges.yaml

```diff
@@ -0,0 +1,27 @@
+externalId: pump_model-populate-lift_station_pumps_edges
+name: pump_model-populate-lift_station_pumps_edges
+destination:
+  dataModel:
+    space: {{model_space}}
+    externalId: {{data_model}}
+    version: '{{view_LiftStation_version}}'
```

Comment:
We are mixing snake_case and CamelCase a bit too much. Should we standardise? I don't have strong opinions beyond the mixing of the two as it becomes superhard to remember. One possibility would be to standardise on CamelCase (i.e. lower first): viewLiftStationVersion alternatively PascalCase, ViewLiftStationVersion.

Reply:
Or just make it simple and do snake_case all over...